.circleci/config.yml

```diff
@@ -72,6 +72,23 @@ jobs:
           paths:
             - ~/.npm
 
+  check_for_manual_release:
+    <<: *container_config
+    steps:
+      - *restore_repo
+      - *sum_lockfiles
+      - *restore_npm_cache
+      - *npm_install
+      - run:
+          name: Check for manual releases
+          command: |
+            set -e
+            if git log --max-count=1 --pretty=%s | grep -q "chore(release): Publish"; then
+              echo "Skipping run on release commit."
+            else
+              npm run preventManualRelease
```

Comment:
Trying to parse, but brain does not compute 😭, what is this command doing here?

Reply:
This is saying that if the commit message is "chore(release): Publish" then don't run this script. The reason for this is we want the publish step (which is handled by lerna) to be able to commit the new versions of each package into master.

Reply:
oooooh I see, probably why the script wasn't running for me when I committed to my branch!

Reply:
Okay, that makes sense!